tests/examples/shims/dbus.lua

```diff
@@ -0,0 +1,24 @@
+local dbus = awesome._shim_fake_class()
+
+function dbus.notify_send(data, appname, replaces_id, icon, title, text, actions, hints, expire)
+    dbus.emit_signal(
+        "org.freedesktop.Notifications",
+        data,
+        appname,
+        replaces_id,
+        icon,
+        title,
+        text,
+        actions,
+        hints,
+        expire
```

Comment:
Use `...`, so it becomes `dbus.emit_sigan("org.freedesktop.Notifications", ...)` ?